src/test/java/org/antonkhmarun/AlertBoxtest.java

```diff
@@ -1,7 +1,7 @@
 package org.antonkhmarun;
```

Comment:
Please add target folder to .gitignore. 
Also please remove those classes from PR